src/Util/OWSAnalytics.h

```diff
@@ -0,0 +1,70 @@
+//
+//  OWSAnalytics.h
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+typedef NS_ENUM(NSUInteger, OWSAnalyticsSeverity) {
```

Comment:
I'm a bit concerned because although I don't have a definitive understanding about this, I've seen some warnings on the internet _not_ to use clang-format as you have it (i.e. only on the portions of a file changed by you) for the reasons I raised yesterday - i.e. formatting can't be done locally.  

But I guess you've been using it this way for a while without problems, so maybe I should just try it.  

Are you using something like this?

https://github.com/andrewseidl/githook-clang-format